Sources/Parsers/AssetsCatalogParser.swift

```diff
@@ -82,7 +133,7 @@ extension AssetsCatalogParser {
   func process(folder: Path, withPrefix prefix: String = "") -> [Catalog.Entry] {
     return (try? folder.children().flatMap {
```

Comment:
@fdiaz I just realised: wouldn't it be simpler instead of all the `Equatable` + `Comparable` implementations you added… to just sort the output of `children()` directly? i.e. apply `folder.children().sorted(by: <).flatMap { … }` to children and not the the final, recursively-constructed structure?

Which would mean in practice removing all the code that you added above and a simpler and mode performant solution in the end, right?

Reply:
Oh wow, you're right. I didn't notice that `Path` was already `Comparable`. I'll just go back with all these changes and add that little change instead. Nice catch!